kore/test/Test/Kore/Builtin/Builtin.hs

```diff
@@ -240,7 +241,7 @@ runStepResultWith solver configuration axiom =
                 (Step.UnificationProcedure Unification.unificationProcedure)
                 [axiom]
                 configuration
-    in runReaderT (SMT.getSMT smt) solver
+    in runReaderT (SMT.getSMT (fmap Foldable.fold <$> smt)) solver
```

Comment:
I noticed a lot of additional calls to `Foldable.fold`. Is there something here which we could give a name to? It's not immediately clear what is being folded.

Reply:
Thanks, I removed the newly added ones.